verification/cbmc/proofs/aws_cryptosdk_priv_hdr_parse_aad/aws_cryptosdk_priv_hdr_parse_aad_harness.c

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <aws/cryptosdk/header.h>
+#include <aws/cryptosdk/private/header.h>
+#include <make_common_data_structures.h>
+#include <proof_helpers/make_common_data_structures.h>
+#include <proof_helpers/proof_allocators.h>
+#include <proof_helpers/utils.h>
+
+void make_hash_table_with_no_backing_store(struct aws_hash_table *map, size_t max_table_entries);
```

Comment:
Can you add a comment as to why this is here? I suppose you are abstracting away this function but then why not remove its function body from the Makefile?

Reply:
Oh thanks for catching that! This used to be a function that I stubbed out but I have since removed the assumption using it from the harness. There's no need for it any more.